extensions/wikia/Search/classes/Test/Controller/SearchControllerTest.php

```diff
@@ -182,84 +182,71 @@ public function testHandleLayoutAbTest() {
 		                    ->disableOriginalConstructor()
-		                    ->setMethods( array( 'getTitle' ) )
+		                    ->setMethods( [ 'getTitle' ] )
 		                    ->getMock();
 		$mockRunHooks = $this->getGlobalFunctionMock( 'wfRunHooks' );
 
 		$originalQuery = 'foo';
 		$redirectUrl = 'http://foo.wikia.com/Wiki/foo';
 
 		$searchConfig
-			->expects	( $this->any() )
-			->method	( 'getPage' )
-			->will		( $this->returnValue( 1 ) )
-		;
+			->expects( $this->any() )
+			->method( 'getPage' )
+			->will( $this->returnValue( 1 ) );
 		$searchConfig
 		    ->expects( $this->any() )
-		    ->method ( 'getQuery' )
-		    ->will   ( $this->returnValue( $mockQuery ) )
-	    ;
+		    ->method( 'getQuery' )
+		    ->will( $this->returnValue( $mockQuery ) );
 		$mockQuery
 		    ->expects( $this->once() )
-		    ->method ( 'getSanitizedQuery' )
-		    ->will   ( $this->returnValue( 'foo' ) )
-		;
+		    ->method( 'getSanitizedQuery' )
+		    ->will( $this->returnValue( 'foo' ) );
 		$searchConfig
 		    ->expects( $this->any() )
-		    ->method ( 'getArticleMatch' )
-		    ->will   ( $this->returnValue( $mockMatch) )
-		;
+		    ->method( 'getArticleMatch' )
+		    ->will( $this->returnValue( $mockMatch) );
 		$mockMatch
 		    ->expects( $this->once() )
-		    ->method ( 'getId' )
-		    ->will   ( $this->returnValue( 123 ) )
-		;
+		    ->method( 'getId' )
+		    ->will( $this->returnValue( 123 ) );
 		$mockArticle
 		    ->expects( $this->once() )
-		    ->method ( 'getTitle' )
-		    ->will   ( $this->returnValue( $mockTitle ) )
-		;
+		    ->method( 'getTitle' )
+		    ->will( $this->returnValue( $mockTitle ) );
 		$searchConfig
-			->expects	( $this->any() )
-			->method	( 'hasArticleMatch' )
-			->will		( $this->returnValue( true ) )
-		;
+			->expects( $this->any() )
+			->method( 'hasArticleMatch' )
+			->will( $this->returnValue( true ) );
 		$mockController
-			->expects	( $this->once() )
-			->method	( 'getVal' )
-			->with		( 'fulltext', '0' )
-			->will		( $this->returnValue( '0' ) )
-		;
+			->expects( $this->any() )
+			->method( 'getVal' )
+			->will( $this->returnValue( '0' ) );
 		$mockRunHooks
-		    ->expects( $this->once() )
-		    ->method ( 'wfRunHooks' )
-		    ->with   ( 'SpecialSearchIsgomatch', array( $mockTitle, $originalQuery ) )
-		;
+			->expects( $this->once() )
+			->method( 'wfRunHooks' );
 		$mockTitle
-			->expects	( $this->any() )
-			->method	( 'getFullURL' )
-			->will		( $this->returnValue( $redirectUrl ) )
-		;
+			->expects( $this->any() )
+			->method( 'getFullURL' )
+			->will( $this->returnValue( $redirectUrl ) );
 		$mockResponse
-			->expects	( $this->once() )
-			->method	( 'redirect' )
-			->with      ( $redirectUrl )
-			->will		( $this->returnValue( true ) )
-		;
+			->expects( $this->once() )
+			->method( 'redirect' )
+			->with( $redirectUrl )
+			->will( $this->returnValue( true ) );
 
 		$responserefl = new ReflectionProperty( 'WikiaSearchController', 'response' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "response" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)